src/languages/nodejs.md

```diff
@@ -7,11 +7,9 @@ Javascript and PHP applications.
 
 ## Supported versions
 
-* 0.12
-* 4.7
 * 4.8
-* 6.9
-* 6.10
+* 6.11
+* 8.2
```

Comment:
In Slack, @GuGuss said we didn't actually remove 6.10.  Therefore we shouldn't remove it from the docs, either.  Let's restore it.